home_work_1_zd_2.js

```diff
@@ -0,0 +1,19 @@
+const MASSNUMS = [4, 7, 9, 2, 8, 4, 10, 15, 6, 19];
+let obj = {};
+
+function get_avg(mass) {
```

Comment:
please follow documented naming for the JS functions

Reply:
Done